services/web/client/source/class/qxapp/component/filter/TagsFilter.js

```diff
@@ -22,30 +22,30 @@ qx.Class.define("qxapp.component.filter.TagsFilter", {
   extend: qxapp.component.filter.UIFilter,
 
   /**
-   * Constructor for TagsFilter creates a workbench TagsFilter.
+   * Constructor for TagsFilter creates a TagsFilter.
    *
    * @extends qxapp.component.filter.UIFilter
    */
-  construct: function(filterId, groupId) {
+  construct: function(label, filterId, groupId) {
     this.base(arguments, filterId, groupId);
     this._setLayout(new qx.ui.layout.HBox());
 
-    this.__dropDown = new qx.ui.toolbar.MenuButton(this.tr("Tags"));
-    this.__dropDown.setMenu(this.__buildMenu());
-    this._add(this.__dropDown);
+    this._dropDown = new qx.ui.toolbar.MenuButton(label);
+    this._add(this._dropDown);
   },
 
   members: {
-    __dropDown: null,
+    _dropDown: null,
```

Comment:
Why from two underscores to only one?

Reply:
I thought it would be useful in derived classes

Reply:
I can't see where else it's used

Reply:
It's not used. But it should be accessible to child classes..